### ISSUE TITLE
FON Fail Erkläring

### ISSUE THREAD

**Issue by espeitslo** [a-sit-plus/at-registrierkassen-mustercode#485]
Hallo,

könnte mir bitte jemand helfen, wir sind beim Einrichtung der StartBelege, beim letzten Versuch bekamm die folgende Meldung mittels Webservice zurück und ich kann daraus nicht bestimmen auls welchen Grund es abgelehnt würde:

<msg>Der übermittelte Beleg ist fehlerhaft.</msg>
		</rkdbMessage>
		<verificationResultList>
			<verificationResult>
				<verificationId>VERIFICATION_FROM_CASHBOX</verificationId>
				<version>1</version>
				<verificationName>Prüfergebnis - Kasse</verificationName>
				<verificationTextualDescription>Bei der Belegprüfung wird untersucht, ob die Vorgaben der RKSV in Bezug auf den maschinenlesbaren Code am Beleg und auf die meldepflichtigen Metadaten der Sicherheitseinrichtung befolgt werden. Im Fehlerfall sind die genauen Fehlerinformationen nachfolgend angeführt.</verificationTextualDescription>
				<verificationState>FAIL</verificationState>
				<verificationResultDetailedMessage>Der vorliegende Beleg ist zwar gesetzeskonform zustande gekommen, die Registrierkasse war aber zum Belegerstellungzeitpunkt  noch nicht mittels Startbelegprüfung in Betrieb genommen. Bitte überprüfen Sie, ob in der Zwischenzeit die erforderliche Startbelegprüfung stattgefunden hat.</verificationResultDetailedMessage>
				<verificationTimestamp>2017-03-24T16:34:11.574+01:00</verificationTimestamp>
				<verificationResultList>
					<verificationResult>
						<verificationId>RECEIPT_FULL</verificationId>
						<version>1</version>
						<verificationName>Detailprüfung des maschinenlesbaren Codes</verificationName>
						<verificationTextualDescription>Dieses Modul und die dazugehörigen Submodule überprüfen die Gültigkeit des Belegs. Dabei werden sowohl Formatprüfungen, kryptographische Prüfungen (verschlüsselter Umsatzzähler, Signatur) als auch Prüfungen im Zusammenhang mit dem Status der Kasse bzw. der Signatur-/Siegelerstellungseinheit durchgeführt.</verificationTextualDescription>
						<verificationState>FAIL</verificationState>
						<verificationTimestamp>2017-03-24T16:34:14.689+01:00</verificationTimestamp>
						<verificationResultList>
							<verificationResult>
								<verificationId>STATE_CONSOLIDATED</verificationId>
								<version>1</version>
								<verificationName>Statusprüfung der Kasse und Signatur-/Siegelerstellungseinheit</verificationName>
								<verificationTextualDescription>**In diesem Modul und den dazugehörigen Submodulen werden die Zustände der Kasse und der verwendeten Signatur-/Siegelerstellungseinheit und die Ergebnisse der zuvor durchgeführten Prüfungen (vor allem der Belegtyp) ausgewertet. Handelt es sich um einen Startbeleg, wird überprüft ob eine Inbetriebnahme der Kasse möglich ist.</verificationTextualDescription**>
								<verificationState>FAIL</verificationState>
								<verificationTimestamp>2017-03-24T16:34:14.735+01:00</verificationTimestamp>
								<verificationResultList>
									<verificationResult>
										<verificationId>STATE_CASHBOX</verificationId>
										<version>1</version>
										<verificationName>Statusprüfung der Kasse</verificationName>
										<verificationTextualDescription>**Es wird der Status der Kasse zum Belegerstellungszeitpunkt und zum Prüfzeitpunkt ausgewertet. Die extrahierten Informationen werden im Zusammenhang mit weiteren Beleginformationen betrachtet.**</verificationTextualDescription>
										<verificationState>FAIL</verificationState>
										<verificationTimestamp>2017-03-24T16:34:14.735+01:00</verificationTimestamp>
									</verificationResult>
								</verificationResultList>
							</verificationResult>
						</verificationResultList>
					</verificationResult>
				</verificationResultList>
			</verificationResult>
		</verificationResultList>
	</result>
</rkdbResponse>

vielen dank im voraus

MfG

**Comment by ErichFreitag** [comment]
Das klingt danach, dass das kein Startbeleg war, den sie zur Prüfung verwendet haben und auch davor noch keiner geprüft wurde.

**Comment by espeitslo** [comment]
Hallo,
ja das ist es so. Wir wissen jetzt nicht genau woran der Fehler liegt, es wurde wirklich nicht als StartBeleg erkannt, aber wie erkennt Finanzonline welches Beleg ein StartBeleg ist? Wie ich weiss ist der Umsatzzähler null und Verkettung enthält die KassenID.

Können Sie mir vielleicht einen Muster XML von einen funktionierenden Startbeleg zukommen lassen damit wir es mit unseren vergleichen können? Bzw womit genau in der XML wird ein StartBeleg überhaupt erkannt? Gibt es ein Flag, oder was anderes?

Vielen dank im voraus

MfG

**Comment by ErichFreitag** [comment]
Die Kriterien für den Startbeleg sind
- Verkettung auf Kassen-ID
- Umsatzzähler auf 0 (AES-Schlüssel korrekt hinterlegt?)
- Barumsätze in den Steuersätzen auf 0

Muster habe ich keines, das würde ihnen auch nichts helfen. Was sagt das A-SIT-Prüftool zu dem Beleg?

**Comment by espeitslo** [comment]
Hallo,
leider ist mir der A-SIT Prüftool nicht bekannt, ich hab zwar überall gelesen dass es ständig verwendet wird, können Sie mich eine kürze Erklärung geben oder besser einen Link mit die Einweisungen?

**Comment by espeitslo** [comment]
Ich weiss wo es zu finden ist, aber es gibt keine "prüftool for dummies" einfache Einleitungen...und leider brennt es schon ziemlich bei uns da 01.04. schon vor die Türe steht

**Comment by ErichFreitag** [comment]
Hier auf der Startseite dieses Forums ist alles beschrieben: https://github.com/a-sit-plus/at-registrierkassen-mustercode#pr%C3%BCftool-100. Sich damit auseinander zu setzen wird nicht erspart bleiben.

**Comment by espeitslo** [comment]
ok das hab ich mir gedacht, ich mach jetzt eine "lange Reschersche" darüber, gibt es das alles vielleicht irgendwo auf englisch da wir eine Ausländische Firma sind?

**Comment by ErichFreitag** [comment]
Nein, bezüglich Prüftool ist mir auf englisch nichts bekannt - so schwierig ist das aber nicht zu verstehen.

**Comment by espeitslo** [comment]
Ok vielen dank, ich werde es einrichten, was hier komisch ist, ist dass wir auch mit einen Funktonierenden StartBeleg von andere Firma versucht haben, logisch - unterschiedlich was nur der Unterschrieft und wenn ich daraus gehe, müsste es doch auch bei uns funktionieren oder?
